CLOSURE 169
java.lang.RuntimeException: Failed to checkout D4J Project: CLOSURE 169
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.nio.file.NoSuchFileException: tmp/Closure-169/buggy/src/com/google/javascript/rhino/jstype/EquivalenceMethod.java
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:370)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:421)
	at java.base/java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:420)
	at java.base/java.nio.file.Files.newInputStream(Files.java:155)
	at java.base/java.nio.file.Files.newBufferedReader(Files.java:2838)
	at java.base/java.nio.file.Files.readAllLines(Files.java:3329)
	at util.PatchDiffUtils.getPatchLineNumbersTarget(PatchDiffUtils.java:59)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:222)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 1
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 1
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 2
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 2
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 3
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 3
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 4
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 4
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 5
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 5
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 6
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 6
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 7
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 7
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 8
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 8
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 9
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 9
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at projects.Defects4JPatch.checkoutD4JProject(Defects4JPatch.java:179)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:46)
	... 1 more
Caused by: java.lang.RuntimeException: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at util.CommandLineRunner.runCommand(CommandLineRunner.java:29)
	at projects.Defects4JPatch.checkoutD4JProject(Defects4JPatch.java:176)
	... 2 more
Caused by: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:166)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:153)
	at util.CommandLineRunner.runCommand(CommandLineRunner.java:27)
	... 3 more
MOCKITO 10
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 10
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at projects.Defects4JPatch.checkoutD4JProject(Defects4JPatch.java:179)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:46)
	... 1 more
Caused by: java.lang.RuntimeException: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at util.CommandLineRunner.runCommand(CommandLineRunner.java:29)
	at projects.Defects4JPatch.checkoutD4JProject(Defects4JPatch.java:176)
	... 2 more
Caused by: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:166)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:153)
	at util.CommandLineRunner.runCommand(CommandLineRunner.java:27)
	... 3 more
MOCKITO 11
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 11
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at projects.Defects4JPatch.checkoutD4JProject(Defects4JPatch.java:179)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:46)
	... 1 more
Caused by: java.lang.RuntimeException: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at util.CommandLineRunner.runCommand(CommandLineRunner.java:29)
	at projects.Defects4JPatch.checkoutD4JProject(Defects4JPatch.java:176)
	... 2 more
Caused by: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:166)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:153)
	at util.CommandLineRunner.runCommand(CommandLineRunner.java:27)
	... 3 more
MOCKITO 18
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 18
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 19
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 19
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 20
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 20
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at projects.Defects4JPatch.lambda$parseFields$0(Defects4JPatch.java:196)
	at util.CommandLineRunner.lineIterator(CommandLineRunner.java:48)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:194)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
MOCKITO 21
java.lang.RuntimeException: Failed to checkout D4J Project: MOCKITO 21
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at projects.Defects4JPatch.checkoutD4JProject(Defects4JPatch.java:179)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:46)
	... 1 more
Caused by: java.lang.RuntimeException: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at util.CommandLineRunner.runCommand(CommandLineRunner.java:29)
	at projects.Defects4JPatch.checkoutD4JProject(Defects4JPatch.java:176)
	... 2 more
Caused by: org.apache.commons.exec.ExecuteException: Process exited with an error: 143 (Exit value: 143)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:166)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:153)
	at util.CommandLineRunner.runCommand(CommandLineRunner.java:27)
	... 3 more
JSOUP 71
java.lang.RuntimeException: Failed to checkout D4J Project: JSOUP 71
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:49)
	at multiedit.coverage.DoCoverageExperimentsBuggy.main(DoCoverageExperimentsBuggy.java:114)
Caused by: java.nio.file.NoSuchFileException: tmp/Jsoup-071/buggy/src/main/java/org/jsoup/nodes/PseudoTextElement.java
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:370)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:421)
	at java.base/java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:420)
	at java.base/java.nio.file.Files.newInputStream(Files.java:155)
	at java.base/java.nio.file.Files.newBufferedReader(Files.java:2838)
	at java.base/java.nio.file.Files.readAllLines(Files.java:3329)
	at util.PatchDiffUtils.getPatchLineNumbersTarget(PatchDiffUtils.java:59)
	at projects.Defects4JPatch.parseFields(Defects4JPatch.java:222)
	at projects.Defects4JPatch.<init>(Defects4JPatch.java:47)
	... 1 more
